liota/__init__.py

```diff
@@ -94,21 +94,21 @@ def setup_package():
             raise IOError('Cannot open config file ' + fullPath)
         dir = os.path.dirname(path)
         log.error('dir: ' + str(dir))
-        if not os.path.exists(dir):
-            log.warn('directory does not exist; create it')
-            try:
-                supermakedirs(dir, 777)
-            except Exception as e:
-                raise OSError('Cannot create directory' + dir)
-            finally:
-                os.umask(original_umask)
-        try:
-            os.mkfifo(path)
-        except OSError as exc:  # Python > 2.5
-            if exc.errno == errno.EEXIST:
-                pass
-            else:
-                raise
+        # if not os.path.exists(dir):
+        #     log.warn('directory does not exist; create it')
+        #     try:
+        #         supermakedirs(dir, 777)
+        #     except Exception as e:
+        #         raise OSError('Cannot create directory' + dir)
+        #     finally:
```

Comment:
As per @bfrggit comment, this code  should be moved from __init__.py to package manager so that we can have the functionality of package manager separate from the normal code. Also the permission should be revisited.